tools/voom_transform/.shed.yml

```diff
@@ -0,0 +1,11 @@
+categories:
+- Statistics
+- RNA
+- Micro-array Analysis
+description: "Voom: Transform count data to fit linear modelling."
+long_description: |
+  Voom: Transform count data to log2-counts per million (logCPM), estimate the mean-variance relationship and use this to compute appropriate observational-level weights.
+name: voom_transform
+owner: yhoogstrate
```

Comment:
@shiltemann what's the policy on ownership? should we use the EMC user for new tools? This tool has no tool shed repository yet, so everything is still possible.